db/migrate/20170103201850_add_has_hs_info_to_protocols.rb

```diff
@@ -0,0 +1,5 @@
+class AddHasHsInfoToProtocols < ActiveRecord::Migration
+  def change
+    add_column :protocols, :has_human_subject_info, :boolean
```

Comment:
This is unnecessary coupling and could lead to data inconsistencies. We already have access to either protocol.research_types_info.human_subjects? or protocol.human_subjects.present?